pkg/core/proxy/integrations/mysql/encode.go

```diff
@@ -14,6 +18,8 @@ import (
 	"go.uber.org/zap"
 )
 
+var reqTimestampMock, responseTimestampMock time.Time
```

Comment:
Why timestamps are made global variables ?

Reply:
they are global because we need to update them in different functions, so we would have to create additional parameters unnecessarily